html/elements/body.json

```
@@ -140,20 +140,14 @@
                 "version_added": true
               },
               "oculus": "mirror",
-              "opera": {
-                "version_added": true
-              },
-              "opera_android": {
-                "version_added": true
-              },
+              "opera": "mirror",
+              "opera_android": "mirror",
               "safari": {
-                "version_added": true
+                "version_added": "1.2"
```

The `bgcolor` attribute is in the initial revision of WebKit:
https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/misc/htmlattrs.in#L13

```suggestion
                "version_added": "1"
```